datacite-doi-commons/src/main/java/no/unit/nva/datacite/DataciteTransformService.java

```diff
@@ -0,0 +1,83 @@
+package no.unit.nva.datacite;
+
+import java.net.URI;
+import java.util.List;
+import java.util.stream.Collectors;
+import javax.xml.bind.JAXBException;
+import no.unit.nva.publication.doi.dto.Contributor;
+import no.unit.nva.publication.doi.dto.Publication;
+import no.unit.nva.publication.doi.dto.PublicationType;
+import no.unit.nva.transformer.Transformer;
+import no.unit.nva.transformer.dto.CreatorDto;
+import no.unit.nva.transformer.dto.DataCiteMetadataDto;
+import no.unit.nva.transformer.dto.IdentifierDto;
+import no.unit.nva.transformer.dto.PublisherDto;
+import no.unit.nva.transformer.dto.ResourceTypeDto;
+import no.unit.nva.transformer.dto.TitleDto;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class DataciteTransformService implements TransformService {
```

Comment:
I removed this service, you should probably update from my branch

Reply:
Removed.